flatten-array.json

```diff
@@ -0,0 +1,29 @@
+{
+   "cases": [
+      {
+         "description": "flattens array with null and integers present",
+         "input": [[]],
+         "expected": []
+      },
+      {
+         "description": "flattens array with just integers present",
+         "input": [1,[2,3,4],5],
+         "expected": [1,2,3,4,5]
+      },
+      {
+         "description": "flattens array with just null",
+         "input": [1,[2,3,4],5,[6,[7,8]]],
+         "expected":[1, 2, 3, 4, 5, 6, 7, 8]
+      },
+      {
+         "description": "flattens array with just null",
+         "input": [0, 2, [[2, 3], 8, 100, 4,[[[50]]]], -2],
+         "expected":[0, 2, 2, 3, 8, 100, 4, 50, -2]
+      },
+      {
+         "description": "flattens array with just null",
```

Comment:
Sorry had problems last night trying to change the pull request from the console, decided to keep it more simple and get rid of null to be more general for all other language